beacon-chain/rpc/prysm/v1alpha1/validator/unblinder.go

```diff
@@ -112,36 +110,35 @@ func (u *unblinder) unblindBuilderBlock(ctx context.Context) (interfaces.SignedB
 		"txs":          len(txs),
 	}).Info("Retrieved full payload from builder")
 
-	bundle, err := unblindBlobsSidecars(u.blobs, blobsBundle)
+	sidecars, err := unblindBlobsSidecars(u.b, blobsBundle)
```

Comment:
should I replace this with buildBlobSidecars function? I would need to add the blobs bundle to the cache, perhaps I add it to the cache when I submit blinded block?